odoo/addons/base/models/ir_attachment.py

```diff
@@ -742,4 +742,4 @@ def regenerate_assets_bundles(self):
             ('res_id', '=', 0),
             ('create_uid', '=', SUPERUSER_ID),
         ]).unlink()
-        self.clear_caches()
+        self.env.registry.clear_cache('asset')
```

Comment:
I made a mistake in my previous suggestion, it is 'assets' with a 's' at the end.

Reply:
Resolved and repushed